frontend/react/src/components/layout/Part.js

```diff
@@ -6,21 +6,22 @@ import QuestionComponent from "../fields/QuestionComponent";
 
 const showPart = (context_data, programType) => {
   if (context_data &&
-      programType &&
-      _.has(context_data, "show_if_state_program_type_in") &&
-      !context_data.show_if_state_program_type_in.includes(programType)) {
+    programType &&
+    _.has(context_data, "show_if_state_program_type_in") &&
+    !context_data.show_if_state_program_type_in.includes(programType)) {
     return false;
   }
   return true;
 }
 
 const Part = ({ partId, text, title, context_data, programType }) => {
   if (showPart(context_data, programType)) {
+    // Determine Part Number from partId
+    let partNum = Number(partId.split("-").pop());
```

Comment:
```suggestion
    let partNum = partId.split("-").pop();
```

I'd probably skip the number conversion, since it doesn't actually matter. It gets used as a string anyway, so I'd be a little wary about some future where there's a part 3B or something, and then this code throws an exception and we have to hunt it down unexpectedly. 😛

Reply:
I used Number to strip off the leading zero. Is there a better way to handle that?

Reply:
Ah, that makes sense!